index.html

```diff
@@ -507,7 +512,89 @@ <h3>StatusList2021Credential</h3>
 <a href="#bitstring-encoding"></a>.
               </td>
             </tr>
-
+            <tr>
+              <td>
+                credentialSubject.ttl
+              </td>
+              <td>
+                The <code>ttl</code> indicates the "time to live" in milliseconds.
+                This property MAY be present, but if not present, implementers MUST 
+                use a value of <code>300000</code> for this property.  A verifier
+                MUST NOT use a cached <code>StatusList2021Credential</code> that was
+                cached for more than the <code>ttl</code> duration prior to the
+                start of verification operation on a <a>verifiable credential</a>.
+                Implementations that publish the status list SHOULD align
+                any protocol-specific caching information, such as the 
+                HTTP `Cache-Control` header, with the value in this field.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialSubject.size
+              </td>
+              <td>
+                The <code>size</code> indicates the size of the status entry in bits.
+                If size is omitted then the size of each status entry is assumed to be
+                one bit (and therefore capable of capturing a boolean status).
+                <code>size</code> MAY be provided, but if <code>size</code> is not present
+                as a property of the <code>credentialStatus</code> then <code>size</code>
+                MUST be processed as `1`.  <code>size</code> MUST be greater than zero.
+                if <code>size</code> is provided, and is greater than `1`, then the property
+                <code>credentialStatus.message</code> MUST be present, and the number of
+                status messages must equal the number of possible values.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                credentialSubject.message
+              </td>
+              <td>
+                The <code>message</code> property MUST be an array, and if present
+                the length of the array must equal the number of possible status states
+                indicated by <code>size</code>. <code>message</code> MAY be present if
+                <code>size</code> is `1`. <code>message</code> MUST be present if
+                <code>size</code> is greater than `1`.  If not present, the message value
+                associated with the bit value of <code>0</code> is "unset" and the bit
+                value of <code>1</code> is "set".
+                Elements in the <code>message</code> array if present, MUST contain at 
+                minimum two properties:
+                <ul>
+                  <li><code>status</code> being a string of the hex value of the status</li>
+                  <li><code>value</code> a string containing the associated message</li>
```

Comment:
Should a statusList that changes its value break invalidate existing credentials?

i.e. if i go from: 
```
"message": [ 
    {"status":"0x00", "value":"valid"}, 
    {"status":"0x01", "value":"invalid"}, 
    {"status":"0x02", "value":"hold"}
]
```
"message": [ 
    {"status":"0x00", "value":"valid"}, 
    {"status":"0x01", "value":"invalid"}, 
    {"status":"0x02", "value":"hold"}
    {"status":"0x03", "value":"example"}
]
```
Or to:
```
"message": [ 
    {"status":"0x00", "value":"valid"}, 
    {"status":"0x01", "value":"invalid"}, 
    {"status":"0x02", "value":"example"}
]
```
we probably want to issue guidance on that case

Reply:
open to suggestions for informative or normative text on that area - i suspect that that will depend on the issuers / status tracking party's use case